public/partials/farmer_profile_large.html

```diff
@@ -98,119 +98,130 @@ <H1 class="farmerSummary">{{farmer.fa_first_name}} {{farmer.fa_middle_name}} {{f
-                            <md-button ng-click="newCommodityItem()" class="md-fab edge" aria-label="Add New Supply">
-                                <md-icon class="edge" md-svg-icon="/images/ic_add_24px.svg"></md-icon>
-                            </md-button>
+                    <div flex layout="row">
+                        <md-icon class="no-margin" md-svg-icon="/images/icons/icons_deceased.svg"></md-icon>
+                        <div layout-align="center" style="text-align: center;" layout="column" ng-class="{'md-offset': 1}">
+                            Available
+                            <div class="label">Status</div>
                         </div>
                     </div>
-                    <div ng-include="newCommodity ? '/partials/commodity_new.html':null">
 
+                </div>
+
+
+                <div layout-align="space-around start" class="farmer_info" layout="row">
+
+
+                    <div flex layout="row">
+                        <md-icon class="no-margin" md-svg-icon="/images/icons/icons_recurring.svg"></md-icon>
+                        <div layout-align="center" style="text-align: center;" layout="column" ng-class="{'md-offset': 1}">
+                            {{isValid(commodity.co_recurring) ? commodity.co_recurring:  'N/A'}}
+                            <div class="label">Recurring</div>
+                        </div>
                     </div>
-                </md-card-content>
-                <md-card-content>
-                    <div class="farmerInfo" layout-padding layout="column" ng-repeat="commodity in commodities">
-                        <div layout-align="space-around start" layout="row">
-                            <div layout-align="left center" layout="column">
-                                {{commodity.cr_crop.cr_crop_name}} - {{commodity.cr_crop.cr_crop_variety}}
-                                <label>Produce/Crop</label>
-                            </div>
-                            <div layout-align="left center" layout="column">
-                                {{commodity.co_quantity}}{{commodity.un_quantity_unit.un_unit_name}}
-                                <label>Amount</label>
-                            </div>
-                            <div layout-align="left center" layout="column">
-                                {{commodity.co_price}} per {{commodity.un_price_unit.un_unit_name}}
-                                <label>Unit Cost</label>
-                            </div>
+
+
+
+                    <div flex layout="row">
+                        <md-icon class="no-margin" md-svg-icon="/images/report-icons/icons_date_approve.svg"></md-icon>
+                        <div layout-align="center" style="text-align: center;" layout="column" ng-class="{'md-offset': 1}">
+                            {{isValid(commodity.co_availability_date) ? getMomentString(commodity.co_availability_date) :  'N/A'}}
```

Comment:
change this to use angular's date filter.  Angular's filters also has support for date formats based on locale.  So if the browser says this is the Caribbean... then it knows that the format is MM/DD/YYYY.  This is one of the nice reasons people use frameworks. We are building on the work done by others.